src/constructs/dns/dns-records.ts

```diff
@@ -13,7 +17,10 @@ export interface GuDnsRecordSetProps {
 }
 
 /**
- * Do not use this construct directly. Use [[`GuCname`]] instead in order to reduce boilerplate.
+ * This construct can be used to create DNS records in NS1.
+ *
+ * If you are creating a CNAME for your CODE and PROD stages, for example when creating a CNAME
+ * for your load balancer, [[`GuCname`]] may be used instead to reduce boilerplate.
```

Comment:
I think we should be more assertive here:

```
Prefer to use [[`GuCname`]] when creating a CNAME for a CODE or PROD load balancer as this requires less boilerplate.
```

Reply:
Good idea, thanks for suggesting this new phrasing 👍